templates/flavors/azure-cni-v1/patches/azure-machine-template.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: AzureMachineTemplate
+metadata:
+  name: ${CLUSTER_NAME}-md-0
+  namespace: default
+spec:
+  template:
+    spec:
+      networkInterfaces:
+      - subnetName: node-subnet-1
+        privateIPConfigs: 109 # 110 max IPConfigs per node
+        acceleratedNetworking: false
```

Comment:
why set `acceleratedNetworking` to false here?

Reply:
It was part of testing since I was experimenting with different VM offering. We can leave it unspecified and azure will take care of enabling it if the VM supports it.

Reply:
Removed `acceleratedNetworking` field.